packages/realm-server/tests/realm-endpoints/lint-test.ts

```diff
@@ -0,0 +1,306 @@
+import { module, test } from 'qunit';
+import supertest, { Test, SuperTest } from 'supertest';
+import { join, resolve, basename } from 'path';
+import { Server } from 'http';
+import { dirSync, setGracefulCleanup, type DirResult } from 'tmp';
+import { copySync, ensureDirSync } from 'fs-extra';
+import {
+  baseRealm,
+  Realm,
+  RealmPermissions,
+  type LooseSingleCardDocument,
+  type QueuePublisher,
+  type QueueRunner,
+} from '@cardstack/runtime-common';
+import {
+  setupCardLogs,
+  setupBaseRealmServer,
+  runTestRealmServer,
+  setupDB,
+  realmServerTestMatrix,
+  realmSecretSeed,
+  createVirtualNetwork,
+  createVirtualNetworkAndLoader,
+  matrixURL,
+  closeServer,
+} from '../helpers';
+import '@cardstack/runtime-common/helpers/code-equality-assertion';
+import { MatrixClient } from '@cardstack/runtime-common/matrix-client';
+import { type PgAdapter } from '@cardstack/postgres';
+import { resetCatalogRealms } from '../../handlers/handle-fetch-catalog-realms';
+
+setGracefulCleanup();
+const testRealmURL = new URL('http://127.0.0.1:4444/');
+const testRealm2URL = new URL('http://127.0.0.1:4445/test/');
+const distDir = resolve(join(__dirname, '..', '..', '..', 'host', 'dist'));
+console.log(`using host dist dir: ${distDir}`);
```

Comment:
I removed this copypasted line in #2370 and also added some test setup helpers, did this come from an old copy of the repository?

Reply:
I pulled this from an old branch. I will update!